RELEASE.md

```diff
@@ -1,33 +1,54 @@
-## Making a release of the Checker Framework Gradle plugin
+## Making and merging pull requests
 
-This document describes how to make a release of the Checker Framework
-Gradle plugin, which this repository contains.
+The Checker Framework Gradle plugin (which this repository contains)
+uses a continuous-delivery style of releases: any time that the
+behavior of the plugin changes, a new release is cut.
 
-### Prequisites
+### Making a pull request
+
+All pull requests should be made from a feature branch. Never push
+directly to `master`.
+
+When you make a pull request that will change any behavior
+of the plugin (including bug fixes and new features), update the version string on 
+your feature branch using these steps:
+1. Make the changes you wish to merge into the plugin.
+2. Choose a new version string. Please try to respect
```

Comment:
This does not *always* have to happen.  It isn't happenening for this pull request, for example.

Reply:
Furthermore, two pull requests may have the same version number.  (This just happened to me, without a merge conflict.)  Therefore, either:

 * the instructions about merging should require double-checking that the version number actually changed, or
 * the version number should be chosen at the time of the merge.